README.md

```diff
@@ -39,6 +39,11 @@ docker run -it -p 8000:8000 opensecurity/mobile-security-framework-mobsf:latest
 ```
 docker run -it -p 8000:8000 -v <your_local_dir>:/root/.MobSF opensecurity/mobile-security-framework-mobsf:latest
 ```
+### For postgres support
```

Comment:
we will move this to docker wiki and add a link to it
`https://github.com/MobSF/Mobile-Security-Framework-MobSF/wiki/7.-Docker-Container-for-MobSF-Static-Analysis `